src/SilverStripeS3AdapterTrait.php

```diff
@@ -11,17 +11,15 @@
 trait SilverStripeS3AdapterTrait
 {
     /**
-     * @return string
      * @throws LogicException
+     *
+     * @return string
      */
     public function findAwsBucket()
     {
         if (getenv('AWS_BUCKET_NAME') !== false) {
             return (string) getenv('AWS_BUCKET_NAME');
         }
-        throw new LogicException(
-            'No valid AWS S3 bucket found. Please set AWS_BUCKET_NAME to your env.'
-        );
+        throw new LogicException('AWS_BUCKET_NAME environment variable not set');
```

Comment:
Just had an annoying thought - shouldn't we leave AWS_ prefix for AWS API constants? Just a though. Annoying because you'd have to change vars in the dash.